mitmproxy broken

**Describe the bug**
A clear and concise description of what the bug is.
mitmproxy tests broken on unstable.
**To Reproduce**
Steps to reproduce the behavior:
nix-env -iA nixos-unstable.mitmproxy

**Additional context**
```
============================= test session starts ==============================
platform linux -- Python 3.7.5, pytest-5.2.1, py-1.8.0, pluggy-0.13.0
rootdir: /build/source, inifile: setup.cfg, testpaths: test
plugins: asyncio-0.10.0
collected 1379 items / 1 errors / 77 deselected / 1301 selected                

==================================== ERRORS ====================================
______________ ERROR collecting test/examples/test_xss_scanner.py ______________
ImportError while importing test module '/build/source/test/examples/test_xss_scanner.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test/examples/test_xss_scanner.py:3: in <module>
    from examples.complex import xss_scanner as xss
E   ModuleNotFoundError: No module named 'examples.complex'
```
**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.2.16, NixOS, 20.03pre201791.c1966522d7d (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre201791.c1966522d7d, nixos-unstable-small-20.03pre201795.d5053d12eb2"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

--- a/pkgs/tools/networking/mitmproxy/default.nix
+++ b/pkgs/tools/networking/mitmproxy/default.nix
@@ -52,10 +52,12 @@ buildPythonPackage rec {
 
   doCheck = (!stdenv.isDarwin);
 
+  # examples.complex.xss_scanner doesn't import correctly with pytest5
   checkPhase = ''
     export HOME=$(mktemp -d)
     export LC_CTYPE=en_US.UTF-8
-    pytest -k 'not test_find_unclaimed_URLs and not test_tcp'
+    pytest --ignore test/examples \
+      -k 'not test_find_unclaimed_URLs and not test_tcp'
   '';
 
   propagatedBuildInputs = [

